js/contextMenus.js

```diff
@@ -1072,6 +1075,18 @@ function onBackButtonHistoryMenu (activeFrame, history, rect) {
         }
       })
     }
+
+    if (stopIndex !== -1) {
+      menuTemplate.push(
+        CommonMenu.separatorMenuItem,
```

Comment:
I think this separator should be added conditionally on if there is at least 1 item before it. I think it's the case there could be 0.